app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md

```diff
@@ -82,3 +87,24 @@ portal_session_conf = {
     "cookie_secure":false
 }
 ```
+
+### Domains
+
+There is a dependency that the dev portal `portal_api_url` and the dev
+portal api `portal_gui_host` need to share a domain or subdomain. The following
```

Comment:
Do you need to say "there is a dependency"? If you simplify to "the dev portal `portal_api_url` and the devportal api `portal_gui_host` need to share a domain or subdomain. " it gets the same message across.

Reply:
No I was just using @satwant17's language. Inc. with some edits:

 The dev portal `portal_api_url` and the dev
portal api `portal_gui_host` must share a domain or subdomain.

Reply:
Done in commit e6da64b